tests/utils.py

```diff
@@ -472,12 +470,16 @@ def create_object(stream):
                 tax_rates=[],  # TODO tax rates
             )
 
-        if stream == 'charges':
+        if stream == 'charges' or stream == 'disputes':
```

Comment:
Why do we need to check differently for these 2 streams? Please explain in the comments.

Reply:
Added in the comment.